src/transport/transporter.js

```diff
@@ -108,6 +103,13 @@ function Transporter(configuration, messageHandler, debugLogging, callback) {
         },
         nextTransport);
     },
+    function (nextTransport) {
+      if (configuration.useWeb3Transport !== true) return nextTransport();
```

Comment:
Web3Transport should be the lowest priority transport if useWeb3Transport isn't set to true, but it should still be used if nothing else is available.  I think this should only skip web3 transport altogether if useWeb3Transport is false AND there's already another transport selected.